[2025-08-03T00:32:24.426+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: comprehensive_analytics_pipeline.wait_for_taxi_data scheduled__2025-08-02T22:25:20.153064+00:00 [queued]>
[2025-08-03T00:32:24.430+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: comprehensive_analytics_pipeline.wait_for_taxi_data scheduled__2025-08-02T22:25:20.153064+00:00 [queued]>
[2025-08-03T00:32:24.431+0000] {taskinstance.py:1359} INFO - Starting attempt 2 of 2
[2025-08-03T00:32:24.438+0000] {taskinstance.py:1380} INFO - Executing <Task(ExternalTaskSensor): wait_for_taxi_data> on 2025-08-02 22:25:20.153064+00:00
[2025-08-03T00:32:24.441+0000] {standard_task_runner.py:57} INFO - Started process 525 to run task
[2025-08-03T00:32:24.443+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'comprehensive_analytics_pipeline', 'wait_for_taxi_data', 'scheduled__2025-08-02T22:25:20.153064+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/comprehensive_analytics_pipeline.py', '--cfg-path', '/tmp/tmpk3e5wfu5']
[2025-08-03T00:32:24.444+0000] {standard_task_runner.py:85} INFO - Job 19: Subtask wait_for_taxi_data
[2025-08-03T00:32:24.479+0000] {task_command.py:415} INFO - Running <TaskInstance: comprehensive_analytics_pipeline.wait_for_taxi_data scheduled__2025-08-02T22:25:20.153064+00:00 [running]> on host 114f606b1815
[2025-08-03T00:32:24.540+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-engineering' AIRFLOW_CTX_DAG_ID='comprehensive_analytics_pipeline' AIRFLOW_CTX_TASK_ID='wait_for_taxi_data' AIRFLOW_CTX_EXECUTION_DATE='2025-08-02T22:25:20.153064+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-08-02T22:25:20.153064+00:00'
[2025-08-03T00:32:24.541+0000] {external_task.py:244} INFO - Poking for tasks ['spark_process_to_iceberg'] in dag nyc_taxi_iceberg_etl on 2025-08-02T22:25:20.153064+00:00 ... 
[2025-08-03T00:33:24.599+0000] {external_task.py:244} INFO - Poking for tasks ['spark_process_to_iceberg'] in dag nyc_taxi_iceberg_etl on 2025-08-02T22:25:20.153064+00:00 ... 
[2025-08-03T00:34:24.658+0000] {external_task.py:244} INFO - Poking for tasks ['spark_process_to_iceberg'] in dag nyc_taxi_iceberg_etl on 2025-08-02T22:25:20.153064+00:00 ... 
[2025-08-03T00:35:24.765+0000] {external_task.py:244} INFO - Poking for tasks ['spark_process_to_iceberg'] in dag nyc_taxi_iceberg_etl on 2025-08-02T22:25:20.153064+00:00 ... 
[2025-08-03T00:36:24.850+0000] {external_task.py:244} INFO - Poking for tasks ['spark_process_to_iceberg'] in dag nyc_taxi_iceberg_etl on 2025-08-02T22:25:20.153064+00:00 ... 
[2025-08-03T00:37:24.938+0000] {external_task.py:244} INFO - Poking for tasks ['spark_process_to_iceberg'] in dag nyc_taxi_iceberg_etl on 2025-08-02T22:25:20.153064+00:00 ... 
[2025-08-03T00:37:24.971+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/external_task.py", line 334, in execute
    super().execute(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 274, in execute
    raise AirflowSensorTimeout(message)
airflow.exceptions.AirflowSensorTimeout: Sensor has timed out; run duration of 300.41627163600015 seconds exceeds the specified timeout of 300.0.
[2025-08-03T00:37:24.982+0000] {taskinstance.py:1398} INFO - Immediate failure requested. Marking task as FAILED. dag_id=comprehensive_analytics_pipeline, task_id=wait_for_taxi_data, execution_date=20250802T222520, start_date=20250803T003224, end_date=20250803T003724
[2025-08-03T00:37:24.991+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 19 for task wait_for_taxi_data (Sensor has timed out; run duration of 300.41627163600015 seconds exceeds the specified timeout of 300.0.; 525)
[2025-08-03T00:37:25.024+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-08-03T00:37:25.055+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
